components/ide-proxy/static/code/marketplace.json

```diff
@@ -17,6 +17,24 @@
           "displayName": "Live Preview"
         }
       },
+      "Shan.code-settings-sync": {
+        "disallowInstall": true,
+        "additionalInfo": "Please use the built-in [Settings Sync](https://aka.ms/vscode-settings-sync-help) functionality instead."
+      },
+      "dbaeumer.jshint": {
+        "disallowInstall": true,
+        "extension": {
+          "id": "dbaeumer.vscode-eslint",
+          "displayName": "ESLint"
+        }
+      },
+      "wwm.better-align": {
+        "disallowInstall": true,
+        "extension": {
+          "id": "Chouzz.vscode-better-align",
+          "displayName": "Better Align"
+        }
+		  },
```

Comment:
Need some :lipstick:

Reply:
Thanks for noticing; I did this change in github.dev, and there it looks fine. Weird